price-feeder/README.md

```diff
@@ -48,6 +48,20 @@ $ price-feeder /path/to/price_feeder_config.toml
 ```
 
 ## Configuration
+There are multiple new options for validators to setup in the price feeder config, the options are telemetry, deviation, and custom endpoints. 
+
+### `Telemetry`
+• A set of options for the /metrics endpoint when running the price feeder. 
+
+### `Deviation`
+• Deviation allows validators to set a custom amount of standard deviations around the median which helps distinguish providers from other providers. 
```

Comment:
Add a note about how the default is for this is 1 standard deviation. 

Also, it's not for distinguishing providers from one another; this "deviation allowance" is supposed to help if any providers became faulty.